index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
Sorry I mean't that the `rbush(data.length)` is faster, even for 100000 features. Not sure why...